examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccountsStorage.agent.ts

```diff
@@ -62,18 +62,18 @@ export class BpiSubjectAccountStorageAgent extends PrismaService {
         creatorBpiSubjectId: bpiSubjectAccount.creatorBpiSubject.id,
         ownerBpiSubjectId: bpiSubjectAccount.ownerBpiSubject.id,
       },
-      include: { OwnerBpiSubject: true, CreatorBpiSubject: true },
+      include: { ownerBpiSubject: true, creatorBpiSubject: true },
     });
 
     return new BpiSubjectAccount(
```

Comment:
now that the prisma model and domain model are in sync, we can directly invoke mapper(newBpiSubjectAccountModel, BpiSubjectAccount,BpiSubjectAccount)